src/org.ts

```diff
@@ -148,10 +148,14 @@ export async function deleteOrg(
   const orgModel = getModelForClass(OrgModelSchema, {
     existingConnection: options.dbConnection,
   });
-  await orgModel.deleteOne({
+  const result = await orgModel.deleteOne({
     name,
   });
 
+  if (result.deletedCount > 0) {
+    options.logger.info({ name }, 'Org deleted');
+  }
```

Comment:
Let's keep things simple and unconditionally log that message.

This `if` is only guarding against a very edge case, where two simultaneous `DELETE /orgs/:orgName` come in for the same org... Which isn't worth the hassle.